src/configuration.cc

```diff
@@ -156,6 +162,25 @@ int Configuration::ParseArguments(int ac, char** av) {
                 << std::endl;
       return -1;
     }
+    if (flags.count("options")) {
+      std::string input_str;
+      const std::vector<std::string> options =
+          flags["options"].as<std::vector<std::string>>();
+      for (const std::string& option: options) {
+        std::vector<std::string> key_value;
+        boost::algorithm::split(key_value, option, boost::is_any_of("="));
```

Comment:
Careful — the value may actually have `=` signs in it, and there's no good way to quote. We should just use `std::string::find` to get the first `=` sign, and leave the rest alone.

Reply:
Done